pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -141,7 +142,7 @@ func add(mgr manager.Manager, c controller.Controller) error {
 
 	for _, secretName := range []string{
 		render.ElasticsearchEksLogForwarderUserSecret,
-		relasticsearch.PublicCertSecret, render.S3FluentdSecretName, render.EksLogForwarderSecret,
```

Comment:
fluentd still talks to elasticsearch, but @vara2504 is working on removing that dependency as we speak so we should be good to keep this change once her PR goes in.